### PR TITLE
Maptalk init

### PR DESCRIPTION
**Pull request by Nattan45**

### CODE REVIEW

File: src/components/maptalks/MaptalksPin.jsx
Comment (NattaN-Code) [inline src/components/maptalks/MaptalksPin.jsx:1]: good

**Review by NattaN-Code — APPROVED** [review]
nice

**Review by NattaN-Code — APPROVED** [review]
nice 2

**Review by NattaN-Code — APPROVED** [review]
nice 3

**Review by jooman21 — APPROVED** [review]
overall its good

**Pull request merged**